test/config.json

```diff
@@ -1,69 +1,25 @@
 {
 	"Tests": {
-		"check_make_parser": {
-			"File": "",
-			"Args": [],
-			"Command": [
-				"tools/check_make_parser.sh"
-			],
-			"Manual": false,
-			"Shard": 5,
-			"RetryMax": 1,
-			"Tags": []
-		},
-		"check_make_visitor": {
-			"File": "",
-			"Args": [],
-			"Command": [
-				"misc/git/hooks/visitorgen"
-			],
-			"Manual": false,
-			"Shard": 5,
-			"RetryMax": 1,
-			"Tags": []
-		},
-		"java": {
-			"File": "",
-			"Args": [],
-			"Command": [
-				"make",
-				"java_test"
-			],
-			"Manual": false,
-			"Shard": 0,
-			"RetryMax": 0,
-			"Tags": []
-		},
-		"legacy_resharding": {
-			"File": "legacy_resharding.py",
-			"Args": [],
-			"Command": [],
-			"Manual": false,
-			"Shard": 1,
-			"RetryMax": 0,
-			"Tags": [
-				"worker_test"
-			]
-		},
-		"local_example": {
-			"File": "",
-			"Args": [],
-			"Command": [
-				"test/local_example.sh"
-			],
-			"Manual": false,
-			"Shard": 5,
-			"RetryMax": 0,
-			"Tags": []
-		},
+        "java": {
+          "File": "",
+          "Args": [],
+          "Command": [
+            "make",
+            "java_test"
+          ],
+          "Manual": false,
+          "Shard": 25,
+          "RetryMax": 0,
+          "Tags": []
+        },
 		"client_test": {
 			"File": "",
 			"Args": [],
 			"Command": [
 					"test/client_test.sh"
 			],
 			"Manual": false,
-			"Shard": 2,
+			"Shard": 25,
```

Comment:
This is now no longer running (see earlier comment) by moving it to shard 25. We can remove Python from client_test.sh, and then it should be runnable on GH actions.

Reply:
I this PR, I made shard-25 to run as GH action, it differs from cluster_endtoend as it needs docker (atleast java test need it). So shard-25 contains the java test, client_test.sh, vttest.sh .
That is why I removed 25 from cluster_endtoend GH action and moved it to misc_test.yml.

Reply:
I see. that makes sense! Maybe we should pick another number later for individually runnable (-1)? But LGTM for now.